### PR TITLE
wip: stream logs using promtail

### PR DESCRIPTION
**Pull request by jakekeeys**

### CODE REVIEW

File: kube-merit-updater [hunk 1]
```diff
@@ -261,14 +263,17 @@ cycle_nodes() {
   nodes=$(retry kubectl --context="${kube_context}" get nodes -l role="${role}",retiring="${retiring}" -o json |\
     jq -r '.items[].metadata.name')
   for node in ${nodes}; do
-    while [[ $(jobs -p | wc -l) -ge $max_nodes ]]; do
+    while [[ $(jobs | grep -v promtail | wc -l) -ge $max_nodes ]]; do
```
Comment (george-angel) [inline kube-merit-updater:266]: I think it would be better to grep positive match if possible

Comment (jakekeeys) [reply kube-merit-updater:266]: agreed will update

Comment (jakekeeys) [reply kube-merit-updater:266]: would the command to grep for be the function name `retry` in this case or `kubectl` not sure how jobs would project it, i would assume the binary aka `kubectl`

**Review by hectorhuertas — APPROVED** [review]

**Review by ffilippopoulos — APPROVED** [review]

File: kube-merit-updater (hunk 1 above)
Comment (george-angel) [reply kube-merit-updater:266]: Actually we are backgrounding all sorts of stuff in that `cycle_node` func, I take back my comment, negative match is better here.

**Review by george-angel — APPROVED** [review]

**Pull request merged**